src/main/webapp/i18n/en/plagiarism.json

```
@@ -57,20 +58,27 @@
                 "plagiarismCases": "Plagiarism Cases",
                 "plagiarismCase": "Plagiarism Case",
                 "resolvedPlagiarismCase": "Resolved Plagiarism Case",
+                "notification": {
+                    "title": "Plagiarism Case: {{ exercise }}",
+                    "body": "Dear {{ student }}, after a meticulous review of your final submission for the exercise “{{ exercise }}“ in the course “{{ course }}“, we have concluded that you have committed plagiarism. This is a violation of the [Student Code of Conduct]({{ cocLink }}) of the faculty of computer science that you have agreed upon. The *[Allgemeine Studien- und Prüfungsordnung]({{ aspoLink }})* (General Examination and Study Regulations) regulates consequences for such cases in §22.1. **You have one week to provide a statement about this situation.**"
+                },
                 "verdict": {
                     "verdict": "Verdict",
-                    "text": "Decide what the consequences of the plagiarism are:",
-                    "verdictDate": "Decided at ",
-                    "verdictBy": "Decided by ",
-                    "none": "No verdict yet",
+                    "text": "Decide what the consequences of the plagiarism are. Your decision can not be reverted.",
```

```suggestion
                    "text": "Decide what the consequences of the plagiarism are. The decision can not be reverted.",
```
My personal preference...